<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kondisi Awal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="logListWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>161</width>
      <height>321</height>
     </rect>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fakta dan Rule yang Terpilih</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stepButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>360</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Satu Iterasi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>360</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Hingga Selesai</string>
    </property>
   </widget>
   <widget class="QListWidget" name="factListWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>151</width>
      <height>171</height>
     </rect>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fakta</string>
    </property>
   </widget>
   <widget class="QListWidget" name="agendaListWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Agenda</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Peta Minesweeper Saat Ini</string>
    </property>
   </widget>
   <widget class="QFrame" name="mapFrame">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>321</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background: rgb(255, 255, 255);
	border: 1px solid;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>321</width>
       <height>321</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="mapGridLayout"/>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
